lib/helpers.js

```diff
@@ -131,3 +132,77 @@ exports.getBaseModelForDataSourceName = function(name, allDataSources) {
   return connectorMeta && connectorMeta.baseModel;
 };
 
+/**
+ * Get the available type choices that should be used for arguments
+ * from workspace.
+ */
+exports.getTypeChoices = function() {
+  var typeChoices = ModelProperty.availableTypes.concat({
+    name: '(other)',
+    value: null
+  });
+  return typeChoices;
+};
+
+/**
+ * Print a sample dummy method implemented in `modelName.js` file
+ * @param {String} modelName Name of model
+ * @param {String} methodName Name of remote method
+ * @param {Boolean} isStatic Method defined on constructor with value true
+ *  while defined on prototype with value false
+ * @param {Array} accepts An array of accept arguments
+ * @param {Array} returns An array of return arguments
+ */
+exports.printDummyMethod = function(
+  modelName,
+  methodName,
+  isStatic,
+  accepts,
+  returns
+) {
+  String.prototype.splice = function(idx, str) {
+    var len = this.length;
+    return this.slice(0, len-idx) + str + this.slice(len-idx);
+  };
+
+  var comma = ', ';
+  var acceptArg = '';
+  var returnArg = '';
+  var functionDef = '= funtion (callback) {' + '\n' +
+  '  //TODO \n' +
+  '  callback(null' + '); \n'+
```

Comment:
Trailing space in both lines above, please remove. I mean the space before `\n` in strings.